.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run lint across different versions of node
+name: Lint CI
+
+on:
+  [push, pull_request]
```

Comment:
It makes no sense to lint on a push as nobody will see/check that. Keep just `pull_request` here